src/components/swap/SwapButtons/SwapButton.tsx

```diff
@@ -24,23 +24,49 @@ const StyledSwapLoadingButton = styled(ButtonPrimary)`
   background-image: linear-gradient(90deg, #4c4c76 19.74%, #292942 120.26%);
   cursor: 'wait';
   padding: 16px;
+
+  ${({ theme }) => theme.mediaWidth.upToExtraSmall`
+    display: grid;
+    gap: 8px;
+    justify-items: center;
+  `};
 `
 
-const StyledPlataformImage = styled.img`
-  margin-top: -3px;
-  margin-right: 6px;
+const RotatingLogo = styled.div`
+  position: relative;
+  display: flex;
+  justify-content: start;
+  align-items: center;
+  width: 112px;
+  height: 22px;
```

Comment:
Should we static set `width` and `height` ?
Consider using `min-height` and `min-width`

Reply:
I'll use `min-width` and `min-height`. Tks!

Reply:
Fixed!